nav2_docking/opennav_docking/src/controller.cpp

```diff
@@ -57,11 +60,13 @@ Controller::Controller(const rclcpp_lifecycle::LifecycleNode::SharedPtr & node)
     std::bind(&Controller::dynamicParametersCallback, this, std::placeholders::_1));
 }
 
-bool Controller::computeVelocityCommand(
-  const geometry_msgs::msg::Pose & pose, geometry_msgs::msg::Twist & cmd, bool backward)
+ bool Controller::computeVelocityCommand(
+    const geometry_msgs::msg::Pose & pose,
+    const geometry_msgs::msg::Pose & robot_pose,
+    geometry_msgs::msg::Twist & cmd, bool backward)
 {
   std::lock_guard<std::mutex> lock(dynamic_params_lock_);
-  cmd = control_law_->calculateRegularVelocity(pose, backward);
+  cmd = control_law_->calculateRegularVelocity(pose, robot_pose, backward);
```

Comment:
Please explain this for me -- is this related to the other item on reverse issues? I'd like @ajtudela's thoughts on this but I don't have a problem with it. I'm just surprised none of us thought about this if it was that simple